eval.py

```diff
@@ -1045,6 +1046,63 @@ def print_maps(all_maps):
     print()
 
 
+def download_file_from_google_drive(id, destination):
```

Comment:
I think it'd be better to keep eval focused on its task (it's already quite large), and could you put this code somewhere into `data/utils.py` or some other suitable path?

Reply:
Sure, I moved it to the existing `utils/functions.py`